circuit_knitting_toolbox/entanglement_forging/entanglement_forging_ground_state_solver.py

```diff
@@ -424,28 +411,40 @@ def evaluate_eigenvalue(parameters: Sequence[float]) -> float:
 
     def get_qubit_operators(
         self,
-        problem: BaseProblem,
-        aux_operators: Optional[
-            ListOrDictType[Union[SecondQuantizedOp, PauliSumOp]]
-        ] = None,
-    ) -> Tuple[PauliSumOp, Optional[ListOrDictType[PauliSumOp]]]:
+        problem: ElectronicStructureProblem,
+    ) -> PauliSumOp:
         """Construct decomposed qubit operators from an ``ElectronicStructureProblem``.
 
         Args:
-          - problem (BaseProblem): A class encoding a problem to be solved.
-          - aux_operators (ListOrDictType[Union[SecondQuantizedOp, PauliSumOp]]): Additional auxiliary operators to evaluate.
+          - problem (ElectronicStructureProblem): A class encoding a problem to be solved.
 
         Returns:
           - hamiltonian_ops: qubit operator representing the decomposed Hamiltonian.
         """
         if not isinstance(problem, ElectronicStructureProblem):
-            raise AttributeError(
+            raise TypeError(
                 "EntanglementForgingGroundStateSolver only supports ElectronicStructureProblem."
             )
-        decomposed_operator = cholesky_decomposition(problem, self._orbitals_to_reduce)
-        hamiltonian_ops = decomposed_operator[0]
-        self._energy_shift = decomposed_operator[1]
 
+        if problem.molecule is None:
```

Comment:
I need to check with @mrossinek to understand how a user can pass an electronic structure problem in the AO basis, and how we can get the mo coeffs in that case.